### ISSUE TITLE
performance benchmarks

### ISSUE THREAD

**Issue by r-cui** [zackhy/TextClassification#14]
Hi zackhy,

May I ask if there are accuracy benchmarks of this project on some datasets like SST2, MR?

I'm asking because I tried running this project on SST2 and got performance not very good. Specifically, by training with the following config, I get accuracy on test set with around 0.7, however I guess the performance of cnn model reported by the original paper is above 0.8
```
python3 train.py \
--data_file=./data/sst2/stsa.binary.train.csv \
--clf=cnn \
--embedding_size=300 \
--num_filters=100 \
--learning_rate 0.0005 \
--batch_size=50 \
--num_epochs=500 \
--evaluate_every_steps=10 \
--save_every_steps=200 \
--num_checkpoint=99999
```
